bin/happy-node-route.py

```diff
@@ -74,6 +74,9 @@
         elif o in ("-e", "--seed"):
             options["seed"] = True
 
+        elif o in ("-y", "--type"):
```

Comment:
Documentation for "type"?

Reply:
Thanks for catching it.
I added in HappyNodeRoute.py

Reply:
done

Reply:
Do we need the distinction for v4 vs v6 route? can it be deduced from the IP address format?

Reply:
Use v4 vs v6, it will make implementation more simple, because the data for route under node is as below:

"route": {
"default_v4": {
"prefix": "10.0.1",
"to": "default",
"via": "onhub"
}
},

If user don't give v4 or v6, it will need extra code to get both v4 and v6 route base on the data, and then base on user need to use IP to sort out v4 or v6.